kqueen/engines/openstack_kubespray.py

```diff
@@ -20,6 +20,9 @@
 logger = logging.getLogger("kqueen_api")
 config = current_config()
 
+MASTER_SECURITY_GR = "kqueen_master"
+COMMON_SECURITY_GR = "kqueen_common"
```

Comment:
what is it? why its hardcoded? if it permanent change, we should document it and provide possibility of change

Reply:
imo we should provide an option on UI , smth like 'create  personal SC' otherwise it should use 'default'
its just one of useful ideas, need to talk about it with u

Reply:
without specific SG, openstack provisioner will not work

Reply:
so need to document it as 'pre-step' or provide kqueen-OPS for that